src/apollos/models/likes/mutations.ts

```diff
@@ -0,0 +1,4 @@
+
+export default [
+  "toggleLike(contentId: String!): [Content]",
```

Comment:
@johnthepink thoughts on both the `toggleLike` and `likes` query returning `[Node]` instead of a `[Content]`. The more I think about it. This would let us use a single collection but extend to all content types.

Instead of the client calling multiple unique queries, they could do something like this:

```js
query likes {
  id
  ... on Content {
     title
     meta { url }
     content { images { s3, cloudfront } } 
  }
  ... on Groups {
    id
    name
    photo
  }
 // etc
}
````